bigtop-packages/src/common/hadoop/do-component-build

```diff
@@ -48,7 +48,28 @@ if [ $HOSTTYPE = "powerpc64le" ] ; then
                 -Dclassifier=linux-ppcle_64 -Dpackaging=exe -Dfile=/usr/src/grpc-java-1.26.0/compiler/build/exe/java_plugin/protoc-gen-grpc-java
 fi
 ## BIGTOP-2288
-
+if [ $HOSTTYPE = "x86_64" ] ; then
+        #download the POWER version of ISA-L
+        yum remove nasm -y
```

Comment:
The do-component-build is used by all supported OS distros including Debian and Ubuntu. You can not assume RHEL variant here.